packages/react/src/patterns/blocks/CalloutWithMedia/CalloutWithMedia.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Callout } from '../../sub-patterns';
+import { ContentBlockSimple } from '../ContentBlockSimple';
```

Comment:
we have to export this component in the `.../packages/react/src/patterns/blocks/index.js` file as well

Reply:
oh yeah updating